jest.config.js

```diff
@@ -15,4 +16,17 @@ module.exports = {
     ),
   },
   testPathIgnorePatterns: ['/node_modules/', 'node_modules_linux'],
+  globals: {
+    // window.config object will now be available in all tests
+    config: {
```

Comment:
So I guess this is now duplicated between index and this config file. Which means we have to maintain it in two places. Is there any idea you have for making a single source of truth for this config info?

Reply:
Not right now.
I tried to extract it to its own module in `lib` folder, but then I think that `scripts/start.sh` wasn't able to find the `config` object, so I reverted it.
We definitely have to find a way to extract this data outside of `index.html`